allennlp/modules/token_embedders/empty_embedder.py

```diff
@@ -0,0 +1,23 @@
+import torch
+from allennlp.modules.token_embedders.token_embedder import TokenEmbedder
+
+
+@TokenEmbedder.register("empty")
+class EmptyEmbedder(TokenEmbedder):
+    """
+    Assumes you want to completely ignore the output of a `TokenIndexer` for some reason, and does
+    not return anything when asked to embed it.
+
+    You should almost never need to use this; normally you would just not use a particular
+    `TokenIndexer`. It's only in very rare cases, like simplicity in data processing for language
+    modeling, where you might want to use this.
```

Comment:
It's actually not exactly that, because you can do what you said above without needing the empty embedder.  It's only because you have a single `TextField` that you have this issue.  You could have split your processing into two separate `TextFields`, and then you wouldn't need this, it just would be slightly messier in other places in the code.  It really is a data processing efficiency issue.

Reply:
I did add a little bit more detail in the docstring, though.

Reply:
Thanks for the detail!